packages/api/blocks-api/src/blocks/block.ts

```diff
@@ -53,10 +53,23 @@ export declare type BlockIndexItem = {
 } & BlockIndexData;
 export declare type BlockIndex = Array<BlockIndexItem>;
 
+// TODO: use from WarningModule after the warning module is in the package (this enum is currently duplicated)
+export enum WarningSeverity {
```

Comment:
have you considered using a union type instead?
`"critical" | "high" | "low"`

Reply:
No, I haven't until now.

I would like to use the same type here as in the [warning.entity.ts](https://github.com/vivid-planet/comet/blob/443fae9654e66e0f0ddc420406f7a0c8621dff88/demo/api/src/warnings/entities/warning.entity.ts#L47) (because as soon as I move everything in the library, i want to remove this enum copy here in the block.ts)

When I use a union type like that in the warning.entity.ts:
"critical" | "high" | "low"

It would also make it a string in the schema, doesn't it? So I think defining it as enum makes sense here or what do you think?